WooCommerce/Classes/ViewModels/CardPresentPayments/CardPresentModalScanningForReader.swift

```diff
@@ -21,6 +21,18 @@ final class CardPresentModalScanningForReader: CardPresentPaymentsModalViewModel
 
     let auxiliaryButtonTitle: String? = nil
 
+    var auxiliaryAttributedButtonTitle: NSAttributedString? {
+        let moreLinkAttributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: UIColor.accent,
+            .underlineStyle: NSUnderlineStyle.single.rawValue
+        ]
+        let moreTextAttributes: [NSAttributedString.Key: Any] = [.foregroundColor: UIColor.text]
+        let linkAttributedString = NSMutableAttributedString(string: Localization.learnMoreLink, attributes: moreLinkAttributes)
+        let moreAttributedString = NSMutableAttributedString(string: Localization.learnMoreText, attributes: moreTextAttributes)
+        linkAttributedString.append(moreAttributedString)
+        return linkAttributedString
```

Comment:
Thanks! I had problems making the link appear following this approach, and was because I didn't add the `%1$@` placeholder to the `NSLocalizedString`. Changed here: 73438f7dbf37b2887841d3b217ff8688c635b7c5